holidays/countries/bolivia.py

```diff
@@ -121,29 +106,21 @@ def _populate(self, year):
 
         # Pando Day.
         if self.subdiv == "N":
-            self[date(year, OCT, 11)] = "Dia del departamento de Pando"
+            self[date(year, OCT, 11)] = "Día del departamento de Pando"
```

Comment:
This one seems to be a bit off - https://en.wikipedia.org/wiki/Pando_Department
Should we change it to Sep 24?

Reply:
Seems Oct 11 - it's Battle of Bahia anniversary, and department holiday is Sep 24. Thank you!